app/assets/javascripts/discourse/app/templates/components/category-title-link.hbs

```diff
@@ -5,6 +5,16 @@
       {{d-icon this.lockIcon}}
     {{/if}}
     <span class="category-name">{{dir-span this.category.name}}</span>
+    {{#if this.site.mobileView}}
+      <div class="category-stat">
+        {{html-safe this.category.statTotal}}
+      </div>
+      {{#unless this.category.pickAll}}
```

Comment:
This looks odd, in some cases we would have two adjacent "category-stat" elements, if I am reading the code correctly?